staff/utils.py

```diff
@@ -806,3 +806,10 @@ def lookup_staff_ids():
         return {person['email']: person['id'] for person in json}
     except(JSONDecodeError, requests.exceptions.MissingSchema):
         return ''
+
+
+def libcal_id_by_email(emailaddr):
+    try:
+        return lookup_staff_ids()[emailaddr]
```

Comment:
Looks like this line is causing the unit tests to fail. It would be easy to solve this by handling for `TypeError` as well but you'll want to make sure this doesn't cause any other problems.

Reply:
Thanks for pointing this out.  I think I have more of a feel for how to interpret the tea leaves of a failed unit test now.

Reply:
I'll handle a `TypeError`, then go in and see whether this noticeably messes anything up on the Wagtail site.

Reply:
(Though I'd like to know how the unit test ended up passing `libcal_id_by_email` an int or whatever it was.)

Reply:
I haven't been able to find any problems with also catching the `TypeError` so far, so I guess I'll go-ahead and push the change up to the branch.